packages/web-app-files/src/mixins/actions/showEditTags.js

```diff
@@ -0,0 +1,46 @@
+import { mapGetters } from 'vuex'
+import { eventBus } from 'web-pkg'
+import { isLocationTrashActive, isLocationPublicActive } from '../../router'
+import isFilesAppActive from './helpers/isFilesAppActive'
+import { SideBarEventTopics } from '../../composables/sideBar'
+
+export default {
+  mixins: [isFilesAppActive],
+  computed: {
+    ...mapGetters(['capabilities']),
+
+    $_showEditTags_items() {
+      return [
+        {
+          name: 'show-edit-tags',
+          icon: 'price-tag-3',
+          label: () => this.$gettext('Add or edit tags'),
+          handler: this.$_showEditTags_trigger,
+          // we don't have details in the trashbin, yet.
```

Comment:
The comment is somewhat true, but looks like it just is a copy-paste-leftover from the `showDetails` file action mixin. Comment can be removed IMO.